packages/web/components/InlineFeedbackPopover/Comment.tsx

```diff
@@ -183,10 +226,10 @@ const Comment: React.FC<CommentProps> = ({ comment, canEdit, onUpdateComment, cu
           </Button>
         </>
       )}
-      {!canEdit && currentUserId && (
-        <div className="edit-block">
+      {!canEdit && currentUser?.id && (
+        <div className={classNames('edit-block', { progress: isLoadingCommentThanks })}>
           <span
-            className="like-btn"
+            className={classNames('like-btn', { 'pointer-events-none': isLoadingCommentThanks })}
             onClick={hasThankedComment ? deleteExistingCommentThanks : createNewCommentThanks}
           >
```

Comment:
Know you didn't do this, but this button should be a button element rather than a span, or at least have role=button if it has to be a span. If you don't mind changing it while we're here that would be awesome, otherwise we can start a misc a11y ticket and add it as a todo there.

Reply:
As a developer, we're frequently forgetting about it. Thanks for this one! Really appreciate this kind of review.

I'll follow with `role=button` to avoid extra CSS code removing button style here. But, it will be great to create another ticket to revisit accessibility in the project. May discuss and define a pattern to follow.